plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/CreateMavenBasedAppEngineStandardProject.java

```diff
@@ -44,6 +45,8 @@ protected void execute(IProgressMonitor monitor)
     SubMonitor progress = SubMonitor.convert(monitor);
     monitor.beginTask("Creating Maven AppEngine archetype", 100);
 
+    // A project id shouldn't be necessary during creation, but the
```

Comment:
Check with @loosebazooka to see if this is likely to be an issue with the new maven plugin. (I suspect not, but I'm not sure.) If it's not, we should add a todo to rip this code out when we move.